src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -732,6 +763,15 @@ internal List<CompletionResult> GetResultHelper(CompletionContext completionCont
                                 case TokenKind.Comma:
                                 case TokenKind.AtParen:
                                     {
+                                        if (lastAst is AssignmentStatementAst assignmentAst && assignmentAst.Left is VariableExpressionAst variableAst)
```

Comment:
> assignmentAst.Left is VariableExpressionAst variableAst

`Left` could be `ConvertExpressionAst` or `AttributeAst`, are we not handling them here?

Reply:
I'll add support and tests